google_cloud_compute/plugin.spec.yaml

```
@@ -337,53 +337,63 @@ connection:
     description: Google Cloud Compute Server
     required: true
     default: https://www.googleapis.com/compute/
+    example: https://www.googleapis.com/compute/
   version:
     type: string
     description: API Version
     required: true
     default: v1
+    example: v1
   project_id:
     title: Project ID
     type: string
     description: Project ID from service credentials
     required: true
+    example: my-project-1
   private_key:
     title: Private Key
     type: credential_asymmetric_key
     description: Private key from service credentials
     required: true
+    example: "-----BEGIN PRIVATE KEY-----MIIEvQIBAFANBgkqhkiG8w0BAQEFAASCBKcwggSjAgEAAoIBAQCpaAjyOCpEqm8Z-----END PRIVATE KEY-----"
   client_email:
     type: string
     description: Client email from service credentials
     required: true
+    example: user@example.com
   client_id:
     title: Client ID
     type: string
     description: Client ID
     required: true
+    example: 109554222673833892525
```

```suggestion
    example: "109554222673833892525"
```
Wrap in quotes to prevent the exponential notation reformatting in the help.md - I think this should fix that.